index.html

```diff
@@ -1097,6 +1100,12 @@ <h4>
             <dd>
               Filters for HID devices.
             </dd>
+            <dt>
+              <dfn>excludes</dfn> member
```

Comment:
Calling this `excludes` seems a little odd to me. `excludes` is a verb and `filters` (at least in my head) is used as a noun. I'd like to rename it to `exclusionFilters` so it's clearer that both attributes hold sets of filters but one set of filters has the opposite effect from the other set.

Reply:
LGTM